tests/containers/forms/CommentsForm.test.jsx

```diff
@@ -73,6 +73,13 @@ describe('<CommentsForm />', () => {
     expect(verifyRedemption).toHaveBeenCalled();
   });
 
+  it('should return null the default case when renderItemDetails is invoked with no selected Item', () => {
+    const instance = shallowWrapper.instance();
+    const selectedItem = {};
+    const result = instance.renderItemDetails(selectedItem);
+    expect(result).toBeNull();
+  });
+
```

Comment:
Commented out code

Reply:
I am going to delete the chunk of code, I think someone had mistakenly left it there.

Reply:
```suggestion
Please remove these dead lines of code if we are not using them
```

Reply:
Please remove the dead chunk of code if we are not using it

Reply:
Thanks for the feedback, I have made the recommended changes.